web-server/js/slycat-color-maps-methods.ts

```diff
@@ -70,6 +75,21 @@ export default {
     return d3.scale.linear().domain(domain).range(this.color_maps[name].colors);
   },
 
+  // Return a d3 version 7 linear color scale with the current color map for the domain [0, 1].
+  // Callers should modify the domain by passing a min and max to suit their own needs.
+  get_color_scale_d3v7: function (name: string, min: number, max: number) {
+    if (name === undefined) name = window.store.getState().colormap;
+    if (min === undefined) min = 0.0;
+    if (max === undefined) max = 1.0;
```

Comment:
```suggestion
    if (!max) max = 1.0;
```

Reply:
This would set max to 1 if the caller passes a max of 0. So I'm tempted to leave the other checks as `=== undefined` as well, to be consistent with this one.